### DISCUSSION TITLE
How to debug TypeLoadException when use emit

### DISCUSSION THREAD

**Discussion by 2A5F** [dotnet/runtime#92421] [Q&A]
`System.TypeLoadException: Could not load type` missing details for why cause this exception

**Comment by huoyaoyuan** [comment]
@dotnet/area-system-reflection-emit The exception message could potentially be improved.

**Comment by 2A5F** [comment]
I fixed it, in my case it is a static field with a value type that has not yet been created in another assembly, the size of type is unknown.
